packages/grafana-ui/src/options/builder/tooltip.tsx

```diff
@@ -42,5 +43,43 @@ export function addTooltipOptions<T extends OptionsWithTooltip>(
       settings: {
         options: sortOptions,
       },
+    })
+    .addRadio({
+      path: 'tooltip.sizing',
+      name: 'Tooltip size',
+      category,
+      settings: {
+        options: [
+          { value: TooltipSizing.Auto, label: 'Auto' },
+          { value: TooltipSizing.Manual, label: 'Manual' },
+        ],
+      },
+      defaultValue: TooltipSizing.Auto,
+    })
+    .addSliderInput({
+      path: 'tooltip.maxTooltipWidth',
+      name: 'Max width',
+      category,
+      description: 'Maximum tooltip width',
+      defaultValue: DEFAULT_TOOLTIP_WIDTH,
+      settings: {
+        min: 0,
```

Comment:
maybe the min value should be the current default min? this might avoid the scrolling issues

Reply:
Good point! But what if the user wants to make it a bit smaller than default? 😁
But sure we can do this, looks like a good protection against layout breaking.

Reply:
i personally like that the old tooltips are nice and compact. it allows them not to hide the data lines unnecessarily.

i would remove any kind of width or min width, and use an implicit maxWidth of what we currently have for width. and allow users to set maxWidth expicitly